recipes/torchtext/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "torchtext" %}
+{% set version = "0.14.1" %}
+# see github.com/conda-forge/conda-forge.github.io/issues/1059 for naming discussion
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pytorch/text/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fd1ef3da7d9c20408c740f7dc7d02ad52a6048b46368355a1a7326d3bc4f2e63
+
+build:
+  number: 0
+  # as of pytorch 1.13, conda-forge only builds for CUDA 11.2+, see
+  # https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/3491
+  skip: true  # [win or cuda_compiler_version in ("None", "10.2", "11.0", "11.1")]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  script:
+    - export FORCE_CUDA="1"    # [linux64 and cuda_compiler_version != 'None']
+    - export CC="$GCC"         # [linux64 and cuda_compiler_version != 'None']
+    - export NVCC_FLAGS="--compiler-bindir=${CC}"    # [linux64 and cuda_compiler_version != 'None']
+    - git submodule update --init --recursive
+    - {{ PYTHON }} -m pip install . -vv
```

Comment:
torchtext has not adopted PEP-518, and I am surprised to see this works.
The official way to build is `python setup.py install`.